civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -47,20 +50,16 @@ def __pull_and_load_requirements(url):
     logger.info('Requirements file ready')
 
 
-def post_save(git_enabled=False):
-
-    def post_save_custom(model, os_path, contents_manager):
-        """ Called from Jupyter post-save hook. Manages save of NB """
+def post_save(model, os_path, contents_manager):
+    """ Called from Jupyter post-save hook. Manages save of NB """
 
-        if model['type'] != 'notebook':
-            return
-        logger.info('Getting URLs to update notebook')
-        update_url, update_preview_url = get_update_urls()
-        if not git_enabled:
-            save_notebook(update_url, os_path)
-        generate_and_save_preview(update_preview_url, os_path)
-        logger.info('Notebook save complete')
-    return post_save_custom
+    if model['type'] != 'notebook':
+        return
+    logger.info('Getting URLs to update notebook')
+    update_url, update_preview_url = get_update_urls()
+    save_notebook(update_url, os_path)
+    generate_and_save_preview(update_preview_url, os_path)
+    logger.info('Notebook save complete')
```

Comment:
This change has change the function from returning a function to returning None. Is that correct? Is Jupyter expecting a closure or did we not need one?

Reply:
we don't need the closure.

Reply:
Ahh cool!